config/locales/en.yml

```diff
@@ -325,9 +325,9 @@ en:
         edit: " Edit Fulfillment"
         delete: " Delete Fulfillment"
       form:
-        date: "Fulfillment Date:"
-        time: "Quantity:"
-        time_unit: "Qty Type:"
+        date: "Fulfillment Date: *"
```

Comment:
Everywhere else in the application we put the '*' directly after the ':'

i.e. Fulfillment Date:*